src/utils.js

```diff
@@ -79,3 +82,26 @@ export function getTypeFromName(typeName: string): void | typeof Value {
       return undefined;
   }
 }
+
+export function describeValue(value: Value): string {
+  let title;
+  let suffix = "";
+  if (value instanceof PrimitiveValue) title = value.toDisplayString();
+  else if (value instanceof ObjectValue) title = "[object]";
```

Comment:
Including the hashValue/object id should be helpful.

Reply:
Done